src/aleph_client/asynchronous.py

```diff
@@ -23,7 +23,8 @@
     AlephMessage,
     AggregateMessage,
     StoreMessage,
-    ProgramMessage, ItemType,
+    ProgramMessage,
+    ItemType,
```

Comment:
Can you separate formatting edits from actual code changes ? Mixing them makes Pull Requests more difficult to process and `git blame` confusing.

Reply:
Done.